joy: {
  assign_x: 0,
  assign_y: 1,
  assign_z: 3,

  max_x: -0.05,
  max_y: 0.05,
  max_z: 0.025,
}

twoWD: {
  first_x: 0,
  first_y: 0,

  radius: 0.1,
  tires_dist: 0.3,
  encoder_cpr: 8192,
}

Arduino: {
  # PIDの順
  gainL: [3.3, 1.0, 0.0],
  gainR: [3.3, 1.0, 0.0],
}